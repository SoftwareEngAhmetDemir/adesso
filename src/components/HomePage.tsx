import Header from './Header';

function HomePage() {
  return (
    <div>
      <Header title="Home Page" />
      <div className="home">
        <img src="/images/beykoz-diamond.jpg" alt="none" />
      </div>
    </div>
  );
}

export default HomePage;
