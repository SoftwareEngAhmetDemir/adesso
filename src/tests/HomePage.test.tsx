import { shallow } from 'enzyme';
import HomePage from '../components/HomePage';

test('should test HomePage component', () => {
 const wrapper = shallow(<HomePage />);
 expect(wrapper).toMatchSnapshot();
});