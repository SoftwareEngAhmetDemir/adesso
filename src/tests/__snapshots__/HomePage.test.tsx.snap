// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test HomePage component 1`] = `
<div>
  <Header
    title="Home Page"
  />
  <div
    className="home"
  >
    <img
      alt="none"
      src="/images/beykoz-diamond.jpg"
    />
  </div>
</div>
`;
